src/scrapers/visa-cal.js

```diff
@@ -145,19 +147,19 @@ function prepareTransactions(txns, startMoment, combineInstallments) {
 
 async function getBankDebits(authHeader, accountId) {
   const bankDebitsUrl = getBankDebitsUrl(accountId);
-  return fetchGet(bankDebitsUrl, authHeader);
+  return fetchGet(bankDebitsUrl, Object.assign(authHeader, HEADER_SITE));
```

Comment:
Can you please revert all 4 occurrences that re-assign `authHeader`: 
1. revert this change so you will pass `authHeader` as is.
2. change line 274 in method `fetchData`  from 
```
    const authHeader = { Authorization: this.authHeader };	
```
 to 
```
    const authHeader = Object.assign({ Authorization: this.authHeader }, HEADER_SITE);
```